### ISSUE TITLE
Is it reasonable to sound if some new feature or extension can be used?

### ISSUE THREAD

**Issue by psionic12** [KhronosGroup/Vulkan-ValidationLayers#5031]
For example, when users using the semaphores, the validate layers can notice the user that there's a new feature called "timeline semaphores", when using synchronizations, notice the "synchronization2", when using descriptor pool, notice that "vk_descriptor_buffer" and so on?

It is really useful in case that Khronos released some new feature which can improve the performance, but a programmer doesn't know.

**Comment by mark-lunarg** [comment]
Best practices is responsible for similar checks such as warning for the use of deprecated/debug/special purpose extensions and functions.  However, the mentioned functionality is a class of warning which does not exist.  I have added this issue as a new [task card in the Best Practices layer project](https://github.com/KhronosGroup/Vulkan-ValidationLayers/projects/1#card-87455976).

**Comment by psionic12** [comment]
The key point is that I'm always afraid of not knowing new extensions which can improve the performance, another example, [VK_EXT_swapchain_maintenance1](https://github.com/KhronosGroup/Vulkan-Docs/blob/main/proposals/VK_EXT_swapchain_maintenance1.adoc) may improve the performance by avoid recreating swapchians when resize window.

**Comment by mark-lunarg** [comment]
@psionic12, yes, we understand and it is a great suggestion! We are tracking it now and it will be prioritized and assigned as resources dictate.  Thank you very much for your consideration.
